MPF.UI.Core/Windows/CreateIRDWindow.xaml.cs

```diff
@@ -0,0 +1,452 @@
+﻿using MPF.Core.UI.ViewModels;
```

Comment:
Nitpick about ordering

Reply:
Do you prefer third-party libraries follow in-built ones?
I think I just let VS sort them for me

Reply:
System, everything else, static in that order. VS should be defaulting to that.